src/main/java/ch/uzh/ifi/hase/soprafs22/controller/DeckController.java

```diff
@@ -87,4 +85,24 @@ public Deck createDeck(@PathVariable("userId") Long userId, @RequestBody DeckPos
         return deck;
     }
 
+    /**
+     * @brief updates the User -> PutMapping
+     *        PUT REQUEST: Status Code 204 -> NO_CONTENT, Error: Status Code = 404
+     *        -> NOT FOUND
+     */
+    @PutMapping("/users/{userId}/decks/{deckId}")
```

Comment:
Do you want to keep this path style? As we discussed with the Cards mapping, we should keep it uniform. What do you think?